etc/Grapl Provision.ipynb

```diff
@@ -198,6 +201,7 @@
     "\n",
     "    provision_master_graph(mclient, schemas)\n",
     "\n",
+    "\n",
```

Comment:
(i also ran `nbqa black` against this file - hence some of the formatting changes - but it is *not* part of our CI)